Add kyverno sample policy

jianh619 opened the pull request.

ycao56 commented on community/CM-Configuration-Management/policy-kyverno-sample.yaml:
```diff
@@ -0,0 +1,71 @@
+metadata:
+  name: policy-kyverno-sample
+  annotations:
+    policy.open-cluster-management.io/standards: NIST-CSF
```
@jianh619 can you update the standard/catogery/control to NIST 800-53 see https://github.com/open-cluster-management/policy-collection/blob/master/community/CM-Configuration-Management/policy-gatekeeper-sample.yaml#L6-L8

ycao56 commented on community/CM-Configuration-Management/policy-kyverno-sample.yaml:
```diff
@@ -0,0 +1,71 @@
+        spec:
+          remediationAction: inform
+          severity: low
+          namespaceSelector:
```
namespaceSelector can be omitted as ClusterPolicy is cluster level resource

ycao56 commented:
Thanks @jianh619 ! This is really great. Just some minor comments. @dockerymick can you review the text?

dockerymick commented on community/README.md:
```diff
@@ -84,6 +84,7 @@ Policy  | Description | Prerequisites
 [ETCD Backup](./CM-Configuration-Management/policy-etcd-backup.yaml) | Use the ETCD Backup policy to receive the last six backup snapshots for etcd. This policy uses the etcd container image in the policy because it contains all required tools like etcdctl. | For more information, see [OpenShift 4 with default storage class](https://docs.openshift.com/container-platform/4.5/storage/dynamic-provis ...
 [Integrity Shield](./CM-Configuration-Management/policy-integrity-shield.yaml)| Use the Integrity Shield to protect the integrity of Kubernetes resources in a cluster (e.g. OpenShift). | See the [Integrity Shield documentation.](https://github.com/open-cluster-management/integrity-shield/blob/master/docs/ACM/README_ENABLE_ISHIELD_PROTECTION_ACM_ENV.md)
 [Integrity Shield Events](./CM-Configuration-Management/policy-integrity-shield-events.yaml)| Use the Integrity Shield Events policy to show a status, which represents whether Integrity Shield has denied some requests in a cluster or not. | See the [Integrity Shield documentation.](https://github.com/open-cluster-management/integrity-shield/blob/master/docs/ACM/README_SHOW_INTEGRITY_STATUS_ON_ACM ...
+[Kyverno sample policy](./CM-Configuration-Management/policy-kyverno-sample.yaml) | Use the Kyverno sample policy to view an example of how kyverno policy can be applied to a managed cluster , this policy will be evaluate by kyverno controller on managed cluster .| See the [Kyverno documentation](https://kyverno.io/docs/writing-policies) for how to write policy. **Note**: Kyverno must be installe ...
```
I suggest the following update:

Use the Kyverno sample policy to view an example of how a kyverno policy can be applied to a managed cluster. This policy is evaluated by the kyverno controller on a managed cluster.



Kyverno must be installed on a managed cluster to use Kyverno policy.

ycao56 replied:
I suggest also add a link to https://kyverno.io/docs/installation/#install-kyverno-using-yamls for installation instructions

dockerymick reviewed (commented):
Thanks for adding the description, I made a comment. Otherwise this looks good

jianh619 commented:
@ycao56 @dockerymick Thanks for your time , just back from holiday , I modified the code following your suggestions and  rebased it to resolve conflict , that's why you saw more comments in this PR .

dockerymick reviewed (approved):
Made a small update, the PR looks good!

ycao56 commented:
@jianh619 your branch seems out of date. There are a lot of file changes unrelated to your pr. Can you rebase?

jianh619 commented:
@ycao56  modified as request , could you help to review please , thanks
